### DISCUSSION TITLE
posts/rails-8-auth/

### DISCUSSION THREAD

**Discussion by giscus** [nithinbekal/nithinbekal.github.io#37] [Comments]
# posts/rails-8-auth/

Nithin Bekal's blog about programming - Ruby, Rails, Vim, Elixir.

https://nithinbekal.com/posts/rails-8-auth/

**Comment by carltonsmith** [comment]
I just assumed the lack of registration controller was just because we are still on release candidates. If there are no plans to include it with 8.0.0 release, that is definitely surprising.

**Reply by nithinbekal** [reply]
Rails 8.0 just dropped, and it looks like it doesn't have registrations. I think the intention was to just have a starting point, and let people build the rest. IMO, signup should be part of that starting point. :(
